integration-tests/src/test/resources/performance-features/cloud-events-performance.feature

```diff
@@ -0,0 +1,101 @@
+#    And Service "webhook-perf-test" with process name "webhook-perf-test" is available within 3 minutes
+#    And Hyperfoil Node scraper is deployed
+#    And Hyperfoil Operator is deployed
+#    And Hyperfoil instance "hf-controller" is deployed within 5 minutes
+
+  @performance
+  Scenario Outline: Sending Cloud Event scenario with usersPerSec <usersPerSec> and maxSessions <maxSessions>
+    When Create benchmark on Hyperfoil "hf-controller" instance with content:
+      """text/vnd.yaml
+      name: rhose-send-cloud-events
+      agents:
+      - agent-one
+      - agent-two
+      http:
+      - host: ${bridge.my-perf-bridge.endpoint.base}
+        sharedConnections: 5
+        connectionStrategy: ALWAYS_NEW
+      phases:
+      - steadyState:
+          constantRate:
+            usersPerSec: <usersPerSec>
+            maxSessions: <maxSessions>
+            startAfter: rampUp
+            duration: 5m
+            maxDuration: 6m
+            scenario:
+            - send-cloud-event:
+              - httpRequest:
+                  POST: ${bridge.my-perf-bridge.endpoint.path}
+                  body: |
+                    {
+                      "specversion": "1.0",
+                      "type": "webhook.site.invoked",
+                      "source": "WebhookActionTestService",
+                      "id": "webhook-test",
+                      "data": {
+                          "bridgeId": "${bridge.my-perf-bridge.id}",
+                          "message": "hello bridge"
+                        }
+                    }
+                  headers:
+                    content-type: application/cloudevents+json
+                    authorization: Bearer ${manager.authentication.token}
+      - rampUp:
+          increasingRate:
+            initialUsersPerSec: 1
+            targetUsersPerSec: 10
+            maxSessions: 10
+            duration: 1m
+            isWarmup: true
+            scenario:
+            - send-cloud-event:
+              - httpRequest:
+                  POST: ${bridge.my-perf-bridge.endpoint.path}
+                  body: |
+                    {
+                      "specversion": "1.0",
+                      "type": "webhook.site.invoked",
+                      "source": "WebhookActionTestService",
+                      "id": "webhook-test",
+                      "data": {
+                          "bridgeId": "${bridge.my-perf-bridge.id}",
+                          "message": "hello bridge"
+                        }
+                    }
+                  headers:
+                    content-type: application/cloudevents+json
+                    authorization: Bearer ${manager.authentication.token}
+      """
+    Then Run benchmark "rhose-send-cloud-events" on Hyperfoil "hf-controller" instance within 15 minutes
+    And number of cloud events sent is greater than 0
+    Examples:
+      | usersPerSec | maxSessions |
```

Comment:
Is it needed to parameterize maxSessions?
I think it can be hardcoded in scenario, what do you think?

Reply:
yes, we can hardcoded for now. Done